Sprint-2/1-key-errors/2.js

```diff
@@ -3,18 +3,24 @@
 
 // this function should square any number but instead we're going to get an error
 
-// =============> write your prediction of the error here
+// =============> write your prediction of the error here 
 
 function square(3) {
     return num * num;
 }
 
 // =============> write the error message here
+Function square(3)
 
 // =============> explain this error message here
-
+// 3 is not approprate parametr name.
+// num is not defined, 
 // Finally, correct the code to fix the problem
 
 // =============> write your new code here
+ function square(num)
+ return num*num 
```

Comment:
This code will look neater using proper syntax. E.g. space between `num*num` and ending the statement with `;`

Also, the function is missing curly braces